Source/Mosa.Compiler.Framework/PhysicalRegisters.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+using System.Collections;
+using System.Diagnostics;
+
+namespace Mosa.Compiler.Framework;
+
+/// <summary>
+/// Physical Registers
+/// </summary>
+public sealed class PhysicalRegisters : IEnumerable<Operand>
+{
+	#region Data Members
+
+	private readonly List<Operand> registers = new();
+
+	#endregion Data Members
+
+	#region Properties
+
+	public int Count => registers.Count;
+
+	public Operand this[int index] => registers[index];
+
+	public bool Is32Platform { get; private set; }
```

Comment:
for consistency this should be `Is32BitPlatform`

Reply:
Good point - also need to fix this same thing elsewhere.